cache.json

```diff
@@ -0,0 +1,6 @@
+{
+  "v11": "cf52b8be2645ee098ee83ea9981fc32a11932fad",
```

Comment:
opinion: either the file should be named more specifically than "cache" or the entries should indicate they're cli repo shas.

Reply:
`npm-cli-latest-version-sha-cache-map.json`?